understanding/22/target-size-minimum.html

```diff
@@ -42,50 +42,47 @@ <h4>Target Size (Minimum)</h4>
+        <p>While the Success Criterion primarily helps touch users in not accidentally triggering adjacent targets, it is also useful for mouse or pen users. It reduces the chances of erroneous activation due to either a tremor or reduced precision, whether because of reduced fine motor control or input imprecision.</p>
       
-        <p>As mobile devices generally have internal heuristics to identify which link is closest, spacing can work as effectively as a larger target size.</p>
+        <h3>Spacing versus size</h3>
+        <p>As mobile devices generally have internal heuristics to identify which link is closest, spacing can work as effectively as a larger target size. The following illustrate a variety of situations. In the top line of icons, the dimensions of each icon target are 44 by 44 CSS pixels with no space in between. This amply passes this Success Criterion and is actually sufficient to meet the AAA requirement for <a href="../21/target-size-enhanced.html">Target Size (Enhanced)</a>. In the next row, the dimensions of each icon target are 24 by 24 CSS pixels with no space in between, passing this Success Criterion. In the third row, the the same icon targets are only 20 by 20 CSS pixels but they have a 4 CSS pixel space in between, passing this Success Criterion via the spacing exception. In the last row, the the same icon targets are only 20 by 20 CSS pixels and have no space in between, failing this Success Criterion.</p>
 
         <figure id="pointer-target-24x24">
             <img src="img/pointer-target-example1-full.png" alt="Four rows of icons with measurements, the first three passing,the last failing the requirement." />
-            <figcaption>In the top line of icons, the dimension of each icon target is 44 by 44 CSS pixels with no space in between, amply passing this Success Criterion. In the next row, the dimension of each icon target is 24 by 24 CSS pixels with no space in between, passing this Success Criterion. In the next row, the the same icon targets are only 20 by 20 CSS pixels but they have 4px space inbetween, passing this Success Criterion. In the last row, the the same icon targets are only 20 by 20 CSS pixels and have no space inbetween, failing this Success Criterion. </figcaption>
+            <figcaption>Four rows of targets, illustrating three ways of meeting this Success Criterion and one way of failing it.</figcaption>
         </figure>
-
+<p>When targets are not 24 by 24 CSS pixels, it is still possible to meet this Success Criteria. The next two illustrations show sets of buttons which are only 20 CSS pixels tall. In the first set, there are no targets immediately above or below the buttons, so they pass. In the second illustration, the buttons have been stacked on top of one another, resulting in a fail.</p>
```

Comment:
"to meet this Success Criteria" > "to meet this Success Criterion". let's not make that criteria/criterion mess again...